rest-api/pom.xml

```diff
@@ -88,6 +88,18 @@
 	</dependencyManagement>
 
 	<dependencies>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger2</artifactId>
+			<version>2.9.2</version>
+		</dependency>
+
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>2.9.2</version>
+		</dependency>
+
```

Comment:
It is unfortunate that springfox is not swagger in that there is a long delay in features. First swagger has to offer features and then springfox has to incorporate them.